app/views/admin/editions/_statistical_data_set_fields.html.erb

```diff
@@ -0,0 +1,20 @@
+<% cache_if edition.statistical_data_set_document_ids.empty?, taggable_statistical_data_sets_cache_digest do %>
```

Comment:
Ditto for this cache key.

```suggestion
<% cache_if edition.statistical_data_set_document_ids.empty?, "#{taggable_statistical_data_sets_cache_digest}-design-system" do %>
```

Reply:
Yea good shout. I wasn't about conflicts while people are using the flag